diff --git a/pytorch/constants.py b/pytorch/constants.py
index ec42825..6efe8ad 100644
--- a/pytorch/constants.py
+++ b/pytorch/constants.py
@@ -32,8 +32,11 @@ GRADIENT_FILE = "gradient.pth"
 GRADIENT_READY_FILE = "gradient_ready.pth"
 WORKER_FINISHED_FILE = "worker_finished.pth"
 
-BASE_DIR = Path(os.getcwd())
-OUTPUT_DIR = Path("/var/tmp/vulkan_trained_models")
+# patched section start
+from tests.constants import TEMP_DIR, TEMP_OUTPUT_DIR
+BASE_DIR = Path(TEMP_DIR)
+OUTPUT_DIR = Path(TEMP_OUTPUT_DIR)
+# patched section end
 
 DATA_PATH = BASE_DIR / "data"
 USER_SCRIPT_PATH = BASE_DIR / "user_script.py"
